### PR TITLE
[Console] Fix PHP 8.1 deprecation in ChoiceQuestion

### LINKED ISSUE #45179: [Console] `PHP 8.1` PHP Deprecated: trim(): Passing null to parameter #1 ($string) of type string is deprecated
### Symfony version(s) affected

6.0.2

### Description

Hi,

With **Symfony/Console 6.0.2** and **PHP 8.1**, there is a depreciation on the method [Symfony\Component\Console\Question\ChoiceQuestion::getDefaultValidator()](https://github.com/symfony/console/blob/v6.0.2/Question/ChoiceQuestion.php#L133) (click on the link to see the line of the issue) : 

```
PHP Deprecated : trim() : Passing null to parameter #1 ($string) of type string is deprecated.
```

This happens when the default value for the question is `null`.

### How to reproduce

```php
// Method definition : https://github.com/symfony/console/blob/v6.0.2/Style/SymfonyStyle.php#L277
$io->choice('Select the queue to analyze', ['queue1', 'queue2', 'queue3']); // The 3rd argument must be `null`.
```

### Possible Solution

We can simply cast the variable to string to fix the issue. Just like this [at this line](https://github.com/symfony/console/blob/v6.0.2/Question/ChoiceQuestion.php#L133):
```diff
-$selectedChoices[$k] = trim($v);
+$selectedChoices[$k] = trim((string) $v);
```

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by ghost**
| Q             | A
| ------------- | ---
| Branch?       | ~6.0~ 4.4 for bug fixes 
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #45179
| License       | MIT
| Doc PR        | symfony/symfony-docs#...

### CODE REVIEW

File: src/Symfony/Component/Console/Tests/Question/ChoiceQuestionTest.php [hunk 1]
```diff
@@ -19,21 +19,21 @@ class ChoiceQuestionTest extends TestCase
...
-        ]);
+            null,
+        ], $default);
 
         $question->setMultiselect($multiSelect);
 
         foreach ($answers as $answer) {
             $validator = $question->getValidator();
             $actual = $validator($answer);
-
```
Comment (chalasr) [inline src/Symfony/Component/Console/Tests/Question/ChoiceQuestionTest.php]: Can you revert this one? Unrelated changes should be avoided

**Review by chalasr — CHANGES_REQUESTED** [review]
This is for 4.4 (my bad - no worry, we can switch the branch while merging).
Thanks for doing your first PR on Symfony!

**Comment by chalasr** [comment]
Can you fix fabbot's report also? (see CI checks)

File: src/Symfony/Component/Console/Tests/Question/ChoiceQuestionTest.php (hunk 1 above)
Comment (ghost) [reply src/Symfony/Component/Console/Tests/Question/ChoiceQuestionTest.php]: Done.

**Comment by ghost** [comment]
> Can you fix fabbot's report also? (see CI checks)

Done too. :)

**Review by chalasr — APPROVED** [review]
For 4.4

**Comment by nicolas-grekas** [comment]
Thank you @BrokenSourceCode.

**Comment by derrabus** [comment]
Linking #41552 to keep track of PHP 8.1 fixes.

**Pull request merged**